File: PRAK504-2510817220020-LailyAzizahIslami.c
#include <stdio.h> 
int reverse(int n){ 
    int hasil = 0;

    while (n != 0) {
        hasil = hasil * 10 + (n % 10);
        n /= 10;
    }

    return hasil;
}
int main() { 
    int A, B; 
    scanf("%d %d",&A,&B); 
    A=reverse(A); 
    B=reverse(B); 
    int C = A+B; 
    printf("%d",reverse(C)); 
} 

File: PRAK504-2510817220020-LailyAzizahIslami.py
def reverse(n):
    hasil = 0

    while (n != 0):
        hasil = hasil * 10 + (n % 10)
        n //= 10
    return hasil

a, b, = map(int , input().split())

a = reverse(a)
b = reverse(b)
c = a + b
d = reverse(c)
print(d)